client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-submit-button.tsx

```diff
@@ -0,0 +1,108 @@
+import { StripeConfiguration } from '@automattic/calypso-stripe';
+import { Button, FormStatus, useFormStatus } from '@automattic/composite-checkout';
+import { useElements, CardElement } from '@stripe/react-stripe-js';
+import { Stripe } from '@stripe/stripe-js';
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import debugFactory from 'debug';
+import { useMemo } from 'react';
+import { State } from 'calypso/state/partner-portal/payment-methods/reducer';
```

Comment:
Minor: this could be `import type` to guarantee it's strippped out by the compiler.

Reply:
Thanks! Addressed in: 430f10bd113e860e5bf646c83a72d8fe3b1cdf9f